NeoML/include/NeoML/Dnn/Layers/TiedEmbeddingsLayer.h

```diff
@@ -33,8 +33,12 @@ class NEOML_API CTiedEmbeddingsLayer : public CBaseLayer {
 	
 	// Embeddings layer name from which we take the matrix.
 	// Only CMultichannelLookupLayer is supported.
-	const char* GetEmbeddingsLayerName() const { return embeddingsLayerName; }
-	void SetEmbeddingsLayerName( const char* name ) { embeddingsLayerName = name; }
+	// Storaged as path with size 1
+	const char* GetEmbeddingsLayerName() const { return embeddingPath.Last(); }
+	void SetEmbeddingsLayerName(const char* name) { embeddingPath = { name }; }
+	// Full path to lookup layer if it is inside of composite
+	const CArray<CString>& GetEmbeddingsLayerPath() const { return embeddingPath; }
+	void SetEmbeddingsLayerPath(const CArray<CString>& path) { path.CopyTo(embeddingPath); }
```

Comment:
Fix comment
"// Embeddings layer name from which we take the matrix.
 // Only CMultichannelLookupLayer is supported.
 // Use this method if the lookupLayer is in the nested level of the dnn (in some nested composite layer)
 "